pom.xml

```diff
@@ -1838,13 +1815,6 @@
         <artifactId>netty</artifactId>
         <version>${netty3.version}</version>
       </dependency>
-      <dependency>
-        <groupId>io.netty</groupId>
-        <artifactId>netty-bom</artifactId>
-        <version>${netty4.version}</version>
-        <type>pom</type>
-        <scope>import</scope>
```

Comment:
Does importing boms work this way? we can get transitivity over a series of imports?

Reply:
Yes it seems to work!

Reply:
Proof added at https://github.com/apache/hbase/pull/6366#issuecomment-2413682833